css/mega-styles.min.css

```diff
@@ -0,0 +1,199 @@
+}
+
+.divider-style span {
+    background: #fff;
+    padding: 0 20px;
+    position: relative;
+    z-index: 5
+}
+
+.divider-style:before {
+    content: "";
+    display: block;
+    border-top: 1px solid #000;
+    width: 100%;
+    height: 1px;
+    position: absolute;
+    top: 50%;
+    z-index: 1
+}
+
+.backdrop {
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    position: absolute;
+    display: block;
+    z-index: 10;
+    visibility: hidden
+}
+
+.backdrop-transition {
+    transition-duration: .2s;
+    transition-timing-function: cubic-bezier(.25, .8, .25, 1);
+    transition-property: visibility
+}
+
+.backdrop-dark {
+    background-color: rgba(0, 0, 0, .8)
+}
+
+.datagrid table tbody .alt td {
+    background: #dfffde;
+    color: #275420
+}
+
+div.dhtmlx_window_active,
+div.dhx_modal_cover_dv {
+    position: fixed!important
+}
+
+.datagrid {
+    font: 12px/150% Arial, Helvetica, sans-serif;
+    background: #fff;
+    overflow: hidden;
+    border: 1px solid #36752d;
+    -webkit-border-radius: 3px;
+    -moz-border-radius: 3px;
+    border-radius: 3px
+}
+
+.datagrid table {
+    border-collapse: collapse;
+    text-align: left;
+    width: 100%
+}
+
+.datagrid table td,
+.datagrid table th {
+    padding: 3px 10px
+}
+
+.datagrid table thead th {
+    background: -webkit-gradient(linear, left top, left bottom, color-stop(.05, #36752d), color-stop(1, #275420));
+    background: -moz-linear-gradient(center top, #36752d 5%, #275420 100%);
+    background-color: #36752d;
+    color: #fff;
+    font-size: 15px;
+    font-weight: 700;
+    border-left: 1px solid #36752d
+}
+
+.datagrid table thead th:first-child {
+    border: none
+}
+
+.datagrid table tbody td {
+    color: #275420;
+    border-left: 1px solid #c6ffc2;
+    font-size: 12px;
+    font-weight: 400
+}
+
+.datagrid table tbody td:first-child {
+    border-left: none
+}
+
+.datagrid table tbody tr:last-child td {
+    border-bottom: none
+}
+
+.datagrid table tfoot td div {
+    border-top: 1px solid #36752d;
+    background: #dfffde;
+    padding: 2px
+}
+
+.datagrid table tfoot td {
+    padding: 0;
+    font-size: 12px
+}
+
+.datagrid table tfoot td ul {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    text-align: right
+}
+
+.datagrid table tfoot li {
+    display: inline
+}
+
+.datagrid table tfoot li a {
+    text-decoration: none;
+    display: inline-block;
+    padding: 2px 8px;
+    margin: 1px;
+    color: #fff;
+    border: 1px solid #36752d;
+    -webkit-border-radius: 3px;
+    -moz-border-radius: 3px;
+    border-radius: 3px;
+    background: -webkit-gradient(linear, left top, left bottom, color-stop(.05, #36752d), color-stop(1, #275420));
+    background: -moz-linear-gradient(center top, #36752d 5%, #275420 100%);
+    background-color: #36752d
+}
+
+.datagrid table tfoot ul a:hover,
+.datagrid table tfoot ul.active {
+    text-decoration: none;
+    border-color: #275420;
+    color: #fff;
+    background: #36752d
+}
```

Comment:
- Consider using @media queries for [responsiveness](https://www.w3schools.com/css/css_rwd_mediaqueries.asp)